test/e2e/edu-1990n-helpers.js

```diff
@@ -0,0 +1,16 @@
+// I really don't want to duplicate this, but the lack of view:veteranId really
```

Comment:
I updated it in my PR to work with 5495, I think we can make it work here: https://github.com/department-of-veterans-affairs/vets-website/pull/5369/files#diff-74108ba937cb81f8e4d99f4fbb4803fc